src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -72,7 +71,7 @@ public void BindException(Exception exception, ISpan span)
         /// <summary>
         /// Returns null.
         /// </summary>
-        public ISpan? GetSpan() => null;
+        public ISpanTracer? GetSpan() => null;
```

Comment:
Ideally we'd keep this `ISpan`. Technically any span can be set to the scope, you can create a child and set to the scope so next startSpan would be two levels down from the transaction

Reply:
`ISpan` doesn't have `StartChild()` though so we do need `ISpanTracer` here tho. Unless you have something else in mind?

Reply:
Can't we add StartChild on `ISpan`? A Span can have children so I don't see a reason not to:
Java has it too: https://github.com/getsentry/sentry-java/blob/7c3f88ad24c95af317c73a042951e1eabc4032be/sentry/src/main/java/io/sentry/Span.java#L67

If we really wanted to split the serialization format from what we have floating around at runtime it would be a reason not to add this. But `ISpanTracer` as a public type is a new concept that wasn't discussed with the wider team.

Reply:
@marandaneto ^

Reply:
yep, sounds like `ISpan` would do the job